__tests__/components/cards/EventCard.test.tsx

```diff
@@ -13,7 +13,6 @@ const testEvent: APIEvent = {
   image: 'URL',
   link: 'URL',
   loc: 'Location',
-  sponsors: [],
   user: 'User',
   attendees_count: 999,
```

Comment:
correct me if im wrong but I seem to remember attendees count being deprecated. For the sake of getting a build quickly we can skip this though

Reply:
Its included in the APIEvent interface so its not optional.